package.json

```diff
@@ -36,8 +36,8 @@
     "standard": "^10.0.2"
   },
   "engines": {
-    "node": ">= 0.8.x",
-    "npm": ">= 1.1.x"
+    "node": ">= 4.8.3",
```

Comment:
Why 4.8.3, and not `>=4`? Just curious if there's something significant about 4.8.3, or if that's the latest 4.8 release.